### ISSUE TITLE
Bookmarks do not update on directives update

### ISSUE THREAD

**Issue by DT3264** [equetzal/huronOS-build-tools#73]
When the bookmarks are updated in the direvtives file, if a browser has already been opened, the bookmarks do not update.

**Comment by equetzal** [comment]
Yeah, this is a very common issue. The main problem is that we should not kill and set default the browser while a contestant is using the browser. Just think of a contesntant sending a problem and just an instant before their browser is closed to update the bookmarks. 
Maybe we can try setting up chromium policies:
Where policies are stored: https://www.chromium.org/administrators/linux-quick-start/
Some definitions vs preferences and policies: https://www.chromium.org/administrators/configuring-other-preferences/
Some troubleshooting about this: https://askubuntu.com/questions/1324149/how-to-configure-policies-preferences-for-chromium
Here is an example of how Chrome manage enterprise policies: https://chromeenterprise.google/policies/?policy=ManagedBookmarks

We'll be needing to test if this feature observe the policies live or only on program start, in which case on every restart stills better that the current state.
I've not researched about Firefox but probably is have something similar. In the case of chrome it won't be possible because it is enterprise feature only, but maybe it's a good idea just stop supporting chrome and do not include it on the build.

**Comment by equetzal** [comment]
Also, as the policies are JSON maybe it would be good to have a util program to do this task (kind of what happly-wallpaper does) but with a more json-friendly language than bash haha. 
It could be python as it is already a dependency of the budgie desktop, or it could be a compiled language and just put the binary to do it.

**Comment by DT3264** [comment]
Or what about creating an extension that somehow retrieves the directives file and uses each browser bookmarks API and given the bookmakrs in the directives file, it only adds the missing ones?
